tests/LocationTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+declare(strict_types=1);
+
+use PHPUnit\Framework\TestCase;
+use base\Location;
+
+final class LocationTest extends TestCase {
+
+    public function testCanBeCreatedWithZeroCoodinate(): void{
+        $this->assertInstanceOf(Location::class, new Location(0.0, 0.0) );
```

Comment:
One unplesant side effect of the constructor throwing errors is the fact that you now have to write this tests.

Reading this test makes me thing you are testing the fact that the constructor in php actually works.

Reply:
In PHP empty string equals to zero if type is not taken into consideration.
Here I'm testing that locations with 0(zero) x y coordinates exists. In system with xOy coordinates this would be in the origin.